lib/systems/src/controllers/DrivebrainController.cpp

```diff
@@ -0,0 +1,74 @@
+#include "controllers/DrivebrainController.h"
+#include "SharedFirmwareTypes.h"
+#include <cstdint>
+
+// #include <iostream>
+
+DrivetrainCommand_s DrivebrainController::evaluate(const VCRData_s &state, unsigned long eval_millis)
```

Comment:
nit: could we rename eval_millis to be curr_millis since that's what most other systems/interfaces are using? just for consistency